.github/workflows/build_rpms.yml

```diff
@@ -0,0 +1,39 @@
+name: RPM Build
+
+on: [pull_request, push]
+
+jobs:
+  rpm_build:
+    name: "Build RPMs"
+    runs-on: ubuntu-latest
+    container: registry.gitlab.com/majorhayden/containers/rpm-builder:latest
+    steps:
+      - name: Prepare container
+        run: |
+          echo "fastestmirror=1" >> /etc/dnf/dnf.conf
+          echo "install_weak_deps=0" >> /etc/dnf/dnf.conf
```

Comment:
I also like `nodocs`, but it triggers lots of warnings in broken spec-files. It does reduce installation size quite a bit, but does not affect package sizes, so this is just FYI..

Reply:
Right, but we're just using these packages to build the RPMs here. We're not making any containers.